src/components/sidebar/helpers/generate-product-landing-nav-items.ts

```diff
@@ -49,19 +50,32 @@ export const generateProductLandingSidebarMenuItems = (
  *
  * Depends on the data located in the associated `src/data/${product.slug}.json`
  * to be loaded into the `product` object passed.
+ *
+ * Can optionally be passed menuItems, which determine sidebar content.
+ * If menuItems are not provided, falls back to a default generated set.
  */
-export const generateProductLandingSidebarNavData = (product: ProductData) => {
+export const generateProductLandingSidebarNavData = (
+  product: ProductData,
+  menuItems?: MenuItem[]
```

Comment:
Accepts optional `menuItems`, if provided, we use that custom content. If not, we fallback to the existing strategy of generating items.